src/components/Product.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import { addMethod } from 'yup';
+import '../styles/Product.css'
+import { Body } from './Body';
+const Product = () => {
+  const [data,setData]=useState({
+    companyname:'',
+    email:'',
+    productname:'',
+    productid:''
+  });
+  const changeHandler= e=>{
+    setData({...data,[e.target.name]:e.target.value})
+  }
+  const submitHandler = e=>{
```

Comment:
Maybe you can improve code for handling submit button , instead of fetching always on click handle submit , you can fetch only once and keep posting data into that fetched API.